pytest-cov/README.rst

```diff
@@ -190,14 +190,20 @@ The terminal report with line numbers::
     TOTAL                  353     20    94%
 
 
-The remaining three reports output to files without showing anything on the terminal (useful for
-when the output is going to a continuous integration server)::
+These three report options output to files without showing anything on the terminal::
 
     py.test --cov-report html
             --cov-report xml
             --cov-report annotate
             --cov myproj tests/
 
+The final report option can also suppress printing to the terminal::
+
+    py.test --cov-report= tests/
```

Comment:
This is no valid command line, please add `--cov-source myproj`. This is the recommended invocation from 2.0 on, `--cov myproj` is deprecated. And while on it, please update it in the example above :)